theme/static/style/style.css

```diff
@@ -248,76 +248,93 @@ pre code {
 
 /* Screenshots Section */
 
-.screenshot-title{
+/* Title */
+.screenshot-title {
     padding: 2% 0;
     text-align: center;
     line-height: 1.4em;
 }
 
-.screenshot-title h3{
+.screenshot-title h3 {
     font-size: 3rem;
 }
 
-.screenshot-title p{
+.screenshot-title p {
     margin-top: 1.5rem;
 }
 
-.img{
+/* Image */
+
+.img-wrapper, .img {
+    width: 100%;
```

Comment:
Make the arrows col-2 width 0, hide them and make the screenshots col-8 width 100%.